Mathlib/GroupTheory/Sylow.lean

```diff
@@ -129,28 +125,28 @@ def comapOfKerIsPGroup (hϕ : IsPGroup p ϕ.ker) (h : ↑P ≤ ϕ.range) : Sylow
       exact (comap_map_eq_self ((P.1.ker_le_comap ϕ).trans hle)).symm }
 
 @[simp]
-theorem coe_comapOfKerIsPGroup (hϕ : IsPGroup p ϕ.ker) (h : ↑P ≤ ϕ.range) :
-    (P.comapOfKerIsPGroup ϕ hϕ h : Subgroup K) = Subgroup.comap ϕ ↑P :=
+theorem coe_comapOfKerIsPGroup (hϕ : IsPGroup p ϕ.ker) (h : P ≤ ϕ.range) :
+    P.comapOfKerIsPGroup ϕ hϕ h = P.comap ϕ :=
```

Comment:
Flagging for reviewers: there are a few instances of removed type ascriptions. I have not thought closely if these need further attention or not.

Reply:
Similarly for removed coercions.